dev-util/gnome-builder/gnome-builder-46.3.ebuild

```diff
@@ -0,0 +1,274 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..12} )
+DISABLE_AUTOFORMATTING=1
+FORCE_PRINT_ELOG=1
+
+inherit gnome.org gnome2-utils llvm meson optfeature python-single-r1 readme.gentoo-r1 virtualx xdg
+
+DESCRIPTION="An IDE for writing GNOME-based software"
+HOMEPAGE="https://wiki.gnome.org/Apps/Builder https://gitlab.gnome.org/GNOME/gnome-builder"
+
+# FIXME: Review licenses at some point
+LICENSE="GPL-3+ GPL-2+ LGPL-3+ LGPL-2+ MIT CC-BY-SA-3.0 CC0-1.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="clang doc +d-spy flatpak +git gtk-doc spell +sysprof test +webkit"
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	flatpak? ( git )
+"
+
+# When bumping, pay attention to all the included plugins/*/meson.build (and other) build files and the requirements within.
+# `grep -rI dependency * --include='meson.build'` can give a good initial idea for external deps and their double checking.
+# The listed RDEPEND order should roughly match that output as well, with toplevel one first then sorted by file path.
+# Most plugins have no extra requirements and default to enabled; we need to handle the ones with extra requirements. Many of
+# them have optional runtime dependencies, for which we try to at least notify the user via DOC_CONTENTS (but not all small
+# things); `grep -rI -e 'command-pattern.*=' -e 'push_arg'` can give a (spammy) idea, plus python imports in try/except.
+
+# Editorconfig needs old pcre, with vte migrating away, might want it optional or ported to pcre2?
+# An introspection USE flag of a dep is required if any introspection based language plugin wants to use it (grep for gi.repository). Last full check at 3.28.4
+
+# TODO: Handle llvm slots via llvm.eclass; see plugins/clang/meson.build
+RDEPEND="
+	>=dev-libs/glib-2.75:2
+	>=gui-libs/gtk-4.10:4[introspection]
+	>=gui-libs/libadwaita-1.5_alpha:1
+	>=gui-libs/libpanel-1.5.0:1
+	>=gui-libs/gtksourceview-5.8:5[introspection]
+	>=dev-libs/json-glib-1.2.0
+	>=dev-libs/jsonrpc-glib-3.43.0:=
+	>=dev-libs/libdex-0.2:=
+	>=dev-libs/libpeas-1.99.0[python,${PYTHON_SINGLE_USEDEP}]
```

Comment:
Please set the right slot for libpeas

Reply:
Done in https://github.com/gentoo/gentoo/pull/34771/commits/2e79d01ce9014f788953f80376df87ba8186e743